pkg/gofr/datasource/file/ftp/fs.go

```diff
@@ -28,6 +29,18 @@ func (c *Conn) RetrFrom(filepath string, offset uint64) (ftpResponse, error) {
 	return c.ServerConn.RetrFrom(filepath, offset)
 }
 
+// List wraps the ftp list method to return a locally defined Entry type.
+func (c *Conn) List(filepath string) ([]Entry, error) {
+	res, err := c.ServerConn.List(filepath)
+
+	var entries []Entry
+	for _, entry := range res {
+		entries = append(entries, entry)
```

Comment:
I thought you had to do this 🤔

```suggestion
		entries = append(entries, Entry(entry))
```

Reply:
Entry is already of type ftp.Entry. Hence type asserting it shows redundant type.